saos-persistence/src/main/java/pl/edu/icm/saos/persistence/model/JudgmentTextContent.java

```diff
@@ -0,0 +1,98 @@
+package pl.edu.icm.saos.persistence.model;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+
+import org.apache.commons.lang3.StringUtils;
+
+import pl.edu.icm.saos.persistence.common.DataObject;
+
+/**
+ * Model class for judgment content
+ * 
+ * @author madryk
+ */
+@Entity
+@Table( uniqueConstraints = { @UniqueConstraint(columnNames = {"fk_judgment"})})
```

Comment:
Good tip. But in this case is not possible. Hibernate won't allow annotation `@Column(unique=true)` on field that is also annotated with `@OneToOne`